core/src/utils.jl

```diff
@@ -1,9 +1,15 @@
 "Get the package version of a given module"
 function pkgversion(m::Module)::VersionNumber
-    rootmodule = Base.moduleroot(m)
-    pkg = Base.PkgId(rootmodule)
-    pkgorigin = get(Base.pkgorigins, pkg, nothing)
-    return pkgorigin.version
+    version = Base.pkgversion(Ribasim)
+    if !isnothing(version)
+        return version
+    else
```

Comment:
I don't know how the Ribasim team is on this, but to have less nesting you could write it as: `!isnothing(version) && return version`

Reply:
Fine by me